### PR TITLE
Truong's pull request week0002 (1st)

### PR DESCRIPTION
**Pull request by truongngodang**
please review code for me. thanks!

### CODE REVIEW

File: week0002/java/week0002.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            col--;
+        }
+
+    }
+
+    private static void displayArray(int[][] array) {
+
+        for (int row = 0; row < array.length; row++) {
+            for (int col = 0; col < array.length; col++) {
+                System.out.print(array[row][col] + "\t");
```
Comment (vietlq) [inline week0002/java/week0002.java]: We do not have any TAB characters in our test file ;)

File: week0002/java/week0002.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            }
+            if (row != array.length - 1) {
+                System.out.print("\n \n");
```
Comment (vietlq) [inline week0002/java/week0002.java]: We don't have such a sequence!

**Comment by truongngodang** [comment]
I fixed code like you talk. Can you review for me?

**Comment by truongngodang** [comment]
move folder like structure standard

File: week0002/truongngodang/java/week0002.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            flag++;
+            row--;
+            col--;
+        }
+
+    }
+
+    private static void writeArrayToFile(int[][] array) {
+        FileOutputStream fos = null;
+        File output = new File("src/output.dat");
```
Comment (vietlq) [inline week0002/truongngodang/java/week0002.java]: Please print straight to STDOUT `(print/println)` and do not create any file

File: week0002/truongngodang/java/output.dat [hunk 4]
```diff
@@ -0,0 +1,4 @@
+ 1  2  3  4
```
Comment (vietlq) [inline week0002/truongngodang/java/output.dat]: Please remove this file. It was not part of submission and not required.

**Comment by vietlq** [comment]
Awesome! You are almost done! See the diff

```
$ diff <(java week0002) test_case_01.dat
4c4
< 10  9  8  7
\ No newline at end of file
---
> 10  9  8  7
```

**Pull request merged**